website/source/docs/job-specification/reschedule.html.md

```diff
@@ -59,6 +61,25 @@ job "docs" {
    number of reschedule happen within it. If more than `attempts` number of
    failures happen with this interval, Nomad will not reschedule any more.
 
+- `delay` `(string: <varies>)` - Specifies the duration to wait before attempting
+  to reschedule a failed task. This is specified using a label suffix like "30s" or "1h".
+
+- `delay_function` `(string: <varies>)` - Specifies the function according to which
+  the initial delay specified in `delay` changes. `delay_function` has three possible
+  values which are described below.
+    - `constant` - The delay between reschedule attempts stays constant at the `delay` value.
+    - `exponential` - The delay between reschedule attempts doubles until it reaches a specified
+      `max_delay` value.
+    - `fibonacci` - The delay between reschedule attempts is calculated by adding the two most recent
```

Comment:
Either say until it reaches a max delay here as well or remove from exponential. I prefer removing from exponential.

Reply:
done